index.js

```diff
@@ -83,7 +76,10 @@ function crossTask(value) {
     localStorage.setItem(TASK_KEY, JSON.stringify(tasks));
 
     renderNumberOfActiveTasks();
+    reloadPage();
+}
 
+function reloadPage() {
```

Comment:
Rename to `reloadTasks`, with this name I thought the page is actually reloading like before